www/nestedexample/README

```diff
@@ -0,0 +1,25 @@
+This plugin permits to create two linked UI elements that can be integrated
```

Comment:
I think this is not clear how the UI element are linked. Please explicit a little bit more.